test/integration/analysis-controller/00-teststep-template.yaml

```diff
@@ -1,19 +1,9 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
+  - script: |
+      envsubst < mock-server.yaml  | kubectl apply -f -
   #  substitutes current time and namespace, making sure they are changed to env var first
   # to prevent bad files in case of a test interrupt
   - script: |
-      current_time=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
-      initial_timestamp=$(date -u -d "$current_time" +%s)
-      seconds_to_add=20
-      new_timestamp=$((initial_timestamp + seconds_to_add))
-      time_20_seconds_later=$(date -u -d "@$new_timestamp" +"%Y-%m-%dT%H:%M:%SZ")
-      export CURRENT_TIME="$current_time"
-      export LATER="$time_20_seconds_later"
-      echo "templating time" $CURRENT_TIME $LATER
-      envsubst <"install.yaml" >out.yaml
-  - command: kubectl apply -f out.yaml
-  #  rewrites the env var not to push useless date changes when running locally
-  - script: |
-      rm out.yaml
+      envsubst < install.yaml | kubectl apply -f -
```

Comment:
Do we actually need envsubst? If I understand correctly, we substitute only namespace variable what in case of we do not specify `.metadata.namespace` in the resources at all, they will be applied to the namespace created by kuttl. Maybe I am missing something, but it should work out of the box when applying everything to the same namespace

Reply:
yes, until https://github.com/keptn/lifecycle-toolkit/issues/2099 is solved, we'll need to do it this way